src/algokit_utils/config.py

```diff
@@ -0,0 +1,22 @@
+from collections.abc import Callable
+
+
+class UpdatableConfig:
```

Comment:
create an singleton instance of this UpdatableConfig within this file. 
So that you can then import the instance directly and toggle the debug mode as needed. That's very similar to what TypeScript version does.

Reply:
If you have issues with pytest you might want to mock or set actual flag on that singletone in conftest file (if such file doesn't exist usually its placed at the root of tests folders where you can define settings applicable for all tests)